b2sdk/file_version.py

```diff
@@ -525,8 +557,8 @@ def from_api_response(self, file_version_dict, force_action=None):
             content_sha1,
             file_info,
             upload_timestamp,
-            file_version_dict['accountId'],
-            file_version_dict['bucketId'],
+            file_version_dict.get('accountId'),
+            file_version_dict.get('bucketId'),
```

Comment:
Now it's less strict. Previous version required `accountId` to be in file_version_dict, current one requires nothing. It's because the method tries to handle two different sets of inputs.

Reply:
Indeed. I decided to change this because I noticed that file_version_dict can have different structure, depending on the exact path we take during the execution. Shall I assume you comment is just an observation or do you expect me to improve something?